nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/RetrieveFromColdStorage.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *  
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.automation.core.operations.document;
+
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.annotations.Param;
+import org.nuxeo.ecm.automation.core.collectors.DocumentModelCollector;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.blob.ColdStorageHelper;
+
+/**
+ * Retrieve the cold storage content associated with the input {@link DocumentModel}.
+ *
+ * @since 11.1
+ */
+@Operation(id = RetrieveFromColdStorage.ID, category = Constants.CAT_BLOB, label = "Retrieve from Cold Storage", description = "Retrieve the cold storage content associated with the document.")
```

Comment:
This should be expanded to explain what "retrieve" means: that's it's a _request_ for retrieval, and is not immediate.

Reply:
i did it on the helper method see [here](https://github.com/nuxeo/nuxeo/blob/feature-NXP-28424-Define-the-ColdStorage-facet/nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java#L105), i will provide more information on java doc of the retrieve operation